spec/unit/provider/registry_key_spec.rb

```diff
@@ -289,6 +297,8 @@
     let(:testval1_wrong_type) { { :name => "one", :type => :string, :data => 255.chr * 1 } }
     let(:testval1_wrong_data) { { :name => "one", :type => :binary, :data => 254.chr * 1 } }
     let(:testval2) { { :name => "two", :type => :binary, :data => 0.chr * 1 } }
+    let(:testval3) { { :name => "three", :type => :binary, :data => 0.chr * 1 } }
+    let(:testval3_data_string) { { :name => "three", :type => :binary, :data => 0.chr * 1 } }
```

Comment:
@btm, yes this binary test case is not doing anything. i can add another context here for dword examples, but this would mean all the test cases(27) run for the dword sample values. currently 54 test cases are run(27 for safe and 27 for unsafe), if i add additional context for dword type, it would mean additional 27 test cases, is that fine with you?